src/Backoffice/Airline/App/Controllers/ListAirlinesController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Airline\App\Controllers;
+
+use Illuminate\Http\JsonResponse;
+use Lightit\Backoffice\Airline\App\Resources\AirlineResource;
+use Lightit\Backoffice\Airline\Domain\Actions\ListAirlinesAction;
+
+class ListAirlinesController
+{
+    public function __invoke(
+        ListAirlinesAction $listAirlinesAction,
+    ): JsonResponse {
+        $airlines = $listAirlinesAction->execute();
+
+        return AirlineResource::collection($airlines)
+            ->response();
```

Comment:
```suggestion
        return AirlineResource::collection($airlines)->response();
```
Consistency with the rest of controllers.